asset/asset.go

```diff
@@ -1730,10 +1730,17 @@ func (a *Asset) deepEqual(allowSegWitIgnoreTxWitness bool, o *Asset) bool {
 		return false
 	}
 
-	if !reflect.DeepEqual(a.ScriptKey, o.ScriptKey) {
+	// If both assets have a tweaked key
+	//
+	// TODO(roasbeef): fin
+	if !a.ScriptKey.PubKey.IsEqual(o.ScriptKey.PubKey) {
 		return false
 	}
 
+	/*if !reflect.DeepEqual(a.ScriptKey, o.ScriptKey) {
```

Comment:
Is this still temporary or should we actually continue just comparing the public key?